web/sites/default/config/field.field.media.document.field_is_accessible.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 63f3e2e8-fd97-403f-8168-765541f711ef
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_is_accessible
+    - media.type.document
+  module:
+    - tmgmt_content
+third_party_settings:
+  tmgmt_content:
+    excluded: false
+id: media.document.field_is_accessible
+field_name: field_is_accessible
+entity_type: media
+bundle: document
+label: 'Meets accessibility requirements'
+description: "Mark all accessibility issues in the document has been remediated.\r\n<ul>\r\n<li>When this checkbox is checked, the Editoria11y module will not display the warning icon where this document is embedded. </li>\r\n<li>More information is available at <a href=\"https://employees.portland.gov/digital-accessibility/create-accessible-documents\">Create accessible documents</a>.</li>\r\n<li>It's the editor's responsibility to make sure any new version of the document remains accessible.</li>\r\n</ul>"
```

Comment:
Typo in description: "Mark [that] all accessibility issues in the document [have] been remediated."

Reply:
Fixed